x/alliance/keeper/reward.go

```diff
@@ -1,6 +1,9 @@
 package keeper
 
 import (
+	"context"
+
+	cmath "cosmossdk.io/math"
```

Comment:
We should follow the math pattern everywhere, that way we know that it refers to the same pacakge.
If you agree change cmath for math in the file 
```suggestion
	"cosmossdk.io/math"
```

Reply:
math is already taken in this file